content/news/2024-02-03-bevy-0.13/index.md

```diff
@@ -38,7 +38,48 @@ TODO.
 
 <div class="release-feature-authors">authors: @TODO</div>
 
-TODO.
+## Transmute Queries
+
+<div class="release-feature-authors">authors: @hymm, james-j-obrien</div>
+
+Have you every wanted to pass a query to a function, but instead of having a
+`Query<&Transform>` you have a `Query<(&Transform, &Velocity)With<Enemy>`?
+Well now you can by using the `Query::transmute_lens` method. Query transmutes
+allow you to change a query into different query types as long as the
+componenets accessed are a subset of the original query.
```

Comment:
If I understand it correctly, it panics if the type is incompatible. I feel like this should be mentioned, it's easy to assume that it would fail to compile instead.